pom.xml

```diff
@@ -526,4 +526,33 @@
 
     </plugins>
   </build>
+
+  <profiles>
+    <profile>
+      <id>nodeps</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>2.19.1</version>
+            <configuration>
+              <runOrder>random</runOrder>
+              <reuseForks>false</reuseForks>
+              <excludes>
+                <exclude>Azure*Test</exclude>
+                <exclude>Amazon*Test</exclude>
+                <exclude>Kakadu*Test</exclude>
+                <exclude>Redis*Test</exclude>
+                <exclude>ControlPanelTest</exclude>
```

Comment:
Sorry, I was thinking of everything with external dependencies, but yeah, that's the complete list of everything with non-free dependencies.

> So do you think we should keep all tests executing by default, have a nodeps profile for developers and users to try (with your list above), and another profile for continuous integration (maybe ci ?) ?

How about: `freedeps` (used with CI), and then either `nodeps` or `alldeps`?

I'm neutral -- whatever you want to go with.